OpenRA.Platforms.Default/Sdl2Input.cs

```diff
@@ -94,6 +94,14 @@ public void PumpInput(Sdl2PlatformWindow device, IInputHandler inputHandler, int
 								case SDL.SDL_WindowEventID.SDL_WINDOWEVENT_SIZE_CHANGED:
 									device.WindowSizeChanged();
 									break;
+
+								case SDL.SDL_WindowEventID.SDL_WINDOWEVENT_MINIMIZED:
+									device.Suspended = true;
+									break;
+
+								case SDL.SDL_WindowEventID.SDL_WINDOWEVENT_RESTORED:
+									device.Suspended = false;
```

Comment:
https://lazyfoo.net/tutorials/SDL/35_window_events/index.php suggests that we should also be doing this for `SDL_WINDOWEVENT_MAXIMIZED`

macOS generates `SDL_WINDOWEVENT_SHOWN` instead of `SDL_WINDOWEVENT_RESTORED` on unminimize, so we probably want to understand exacly what the use cases of `SHOWN`/`HIDDEN`/`EXPOSED` are and hook them up too.